Monopoly/Cards/Deck.cs

```diff
@@ -18,7 +13,7 @@ public Deck(List<ICard> cards)
 
         public virtual ICard Draw()
         {
-            return cards.Dequeue();
+            return cards.Count > 0 ? cards.Dequeue() : null;

```

Comment:
I'd be OK with an exception here.  Not having cards in the deck is not a thing in monopoly. At most one card can plausibly be removed from the deck in the course of normal play.